TreeMaker/python/makeTreeFromMiniAOD_cff.py

```diff
@@ -97,9 +97,12 @@ def makeTreeFromMiniAOD(self,process):
     ## PDF weights for PDF systematics
     ## ----------------------------------------------------------------------------------------------
     if self.geninfo and self.doPDFs:
-        process.PDFWeights = cms.EDProducer('PDFWeightProducer')
-        self.VectorDouble.extend(['PDFWeights:PDFweights','PDFWeights:ScaleWeights'])
-        self.VectorInt.extend(['PDFWeights:PDFids'])
```

Comment:
What happened to this line? Is it not needed anymore?

Reply:
Look below, we're now importing it from a cfi file (to keep track of the parameters).

Reply:
I meant the saving of the PDFids. Is saving that not important anymore? Is that the same as the pdfSetName? Is see nothing in the cfi that says "ID" to me.

Reply:
Ah, no, I don't think it matters. The ID was basically just the index of the weight in the vector, so we don't really need to keep that info separately.